pgjdbc/src/test/java/org/postgresql/test/jdbc2/DatabaseMetaDataTest.java

```diff
@@ -49,88 +50,106 @@
  */
 public class DatabaseMetaDataTest {
   private Connection con;
+  private BinaryMode binaryMode;
+  private boolean sqlTypesWithTimezone;
 
+  public void initDatabaseMetaDataTest(BinaryMode binaryMode, boolean sqlTypesWithTimezone) throws SQLException {
+    final Properties props = new Properties();
+    if (sqlTypesWithTimezone) {
+      PGProperty.USE_WITH_TIMEZONE.set(props, true);
+    }
+    if (binaryMode == BinaryMode.FORCE) {
+      PGProperty.PREPARE_THRESHOLD.set(props, -1);
+    }
+    con = TestUtil.openDB(props);
+
+    this.binaryMode = binaryMode;
+    this.sqlTypesWithTimezone = sqlTypesWithTimezone;
+  }
```

Comment:
The method has been removed in 4956902c7d3a5217b89d68fafa7fcb6a36e070af
Could you please clarify why do you re-add it back?

Reply:
Ah, ok, my mistake. I should have been more careful rebasing. Let me see the easiest way to undo this.

Reply:
Fixed